Source/WebCore/css/mathml.css

```diff
@@ -175,8 +175,9 @@ mphantom {
 }
 
 /* This is a special style for erroneous markup. */
+/* Web Specification: https://w3c.github.io/mathml-core/#error-message-merror */
 merror {
-    outline: solid thin red;
+    border: 1px solid red;
     font-weight: bold;
     font-family: sans-serif;
```

Comment:
Is it possible to remove `font-weight` and `font-family` too?

I guess this can be tested in `merror-001.html` by comparing against the `mrow`'s style?

Reply:
I did and now pushed another EWS run and there might be more failures but I will fix them in later push. :-)